<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cinema</class>
 <widget class="QMainWindow" name="Cinema">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cinema</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>890</width>
      <height>710</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border: 2px solid color: rgb(0, 0, 0);
border radius: 20px;
background-color: rgb(148, 33, 146);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="buddy">
     <cstring>label</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>680</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Andale Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Cinema</string>
    </property>
   </widget>
   <widget class="QLabel" name="Oppenheimer">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>181</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Napolean">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>181</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Spider">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>181</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOPP">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>191</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNAP">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>191</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSPI">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>181</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ReserveOppenhemier">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>520</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 33, 146);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>  Reserve Your Seat</string>
    </property>
   </widget>
   <widget class="QLabel" name="ReserveNapolean">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>520</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 33, 146);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>  Reserve Your Seat</string>
    </property>
   </widget>
   <widget class="QLabel" name="ReserveSpider">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>520</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 33, 146);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>  Reserve Your Seat</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelContrib">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>609</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;YOOO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 253, 255);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Developed by: Aly Elaswad - Adham Mohamed - Laila Sayed - Yousef El Sebaiy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonTrailerOPP">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>560</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 33, 146);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Watch Trailer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonTrailerNAP">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>560</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 33, 146);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Watch Trailer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonTrailerSPI">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>560</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 33, 146);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Watch Trailer</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxSignIN">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>481</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 72, 72);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_SignIN">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>221</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(156, 25, 117);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>Sign in</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>331</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(156, 25, 117);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>      Welcome To the KinoScope</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_UserName_2">
        <property name="styleSheet">
         <string notr="true">
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_UserName_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_Password_2">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Password_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="VerifyLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="NoAccount">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(156, 25, 117);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>Sign Up?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ShowPass_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>180</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(72, 72, 72);</string>
     </property>
     <property name="text">
      <string>👀</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSignUP">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>481</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 72, 72);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_SignUP">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>221</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(156, 25, 117);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>331</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(156, 25, 117);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>      Welcome To the KinoScope</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_UserName">
        <property name="styleSheet">
         <string notr="true">
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_UserName">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_Password">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Password">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="ShowPass">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>200</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(72, 72, 72);</string>
     </property>
     <property name="text">
      <string>👀</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxHello">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>481</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 72, 72);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>331</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(156, 25, 117);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>      Welcome To the KinoScope</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>266</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Already have an account?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonIN">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(156, 25, 117);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>Sign in</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>261</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>New to the KinoScope?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonUP">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(156, 25, 117);
border-color: rgb(0, 0, 0);
border: 2px;
padding-left: 8px;
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>Sign up</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_2</zorder>
   <zorder>Oppenheimer</zorder>
   <zorder>Spider</zorder>
   <zorder>ReserveNapolean</zorder>
   <zorder>ReserveOppenhemier</zorder>
   <zorder>ReserveSpider</zorder>
   <zorder>Napolean</zorder>
   <zorder>labelContrib</zorder>
   <zorder>ButtonTrailerOPP</zorder>
   <zorder>ButtonTrailerNAP</zorder>
   <zorder>ButtonTrailerSPI</zorder>
   <zorder>groupBoxSignIN</zorder>
   <zorder>groupBoxSignUP</zorder>
   <zorder>groupBoxHello</zorder>
   <zorder>pushButtonOPP</zorder>
   <zorder>pushButtonNAP</zorder>
   <zorder>pushButtonSPI</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
